cluster/gce/configure-vm.sh

```diff
@@ -949,7 +949,7 @@ EOF
   if [[ ! -z "${KUBELET_APISERVER:-}" ]] && [[ ! -z "${KUBELET_CERT:-}" ]] && [[ ! -z "${KUBELET_KEY:-}" ]]; then
     cat <<EOF >>/etc/salt/minion.d/grains.conf
   kubelet_api_servers: '${KUBELET_APISERVER}'
-  cbr-cidr: 10.123.45.0/30
+  cbr-cidr: 10.123.45.0/29
```

Comment:
I'm not clear on why this has to change. This case is for when the master is part of the cluster (e.g. the CIDR should start as a /30 and then get changed to a /24 CIDR allocated by the node controller). The block below is for a master that isn't registered and allocates `10.246.0.0/24` by default.